modules/ROOT/pages/faq.adoc

```diff
@@ -1,9 +1,9 @@
 = Fedora CoreOS Frequently Asked Questions
 
 If you have other questions than are mentioned here or want to discuss
-further, join us in our Libera.Chat IRC channel,
-link:ircs://irc.libera.chat:6697/#fedora-coreos[#fedora-coreos], or on our
-https://discussion.fedoraproject.org/c/server/coreos[discussion board].
+further, join us in our Matrix room,
+link:https://chat.fedoraproject.org/#/room/#coreos:fedoraproject.org[#coreos:fedoraproject.org],
+or on our https://discussion.fedoraproject.org/c/server/coreos[discussion board].
```

Comment:
https://discussion.fedoraproject.org/tag/coreos

Reply:
Fixed in a separate commit!